src/SpectatorView.Unity/Assets/SpectatorView.Editor/Scripts/CompositorWindow.cs

```diff
@@ -155,13 +155,34 @@ private void CompositeGUI()
                                 .ToArray());
 
                             compositionManager.CaptureDevice = supportedDevices[selectedIndex];
+
+                            if (supportedDevices[selectedIndex] == FrameProviderDeviceType.AzureKinect)
+                            {
+                                EditorGUILayout.EndHorizontal();
+                                EditorGUILayout.Space();
+                                EditorGUILayout.BeginHorizontal();
+
+                                GUIContent occlusionLabel = new GUIContent("Occlusion Mode", "The occlusion mode used for compositing.");
+
+                                var occlusionModes = Enum.GetValues(typeof(OcclusionSetting))
+                                    .Cast<OcclusionSetting>()
+                                    .ToList();
+
+                                selectedIndex = occlusionModes.IndexOf(compositionManager.OcclusionMode);
+                                selectedIndex = EditorGUILayout.Popup(occlusionLabel, selectedIndex,
+                                     occlusionModes
+                                    .Select(x => x.ToString())
+                                    .ToArray());
+
+                                compositionManager.OcclusionMode = occlusionModes[selectedIndex];
```

Comment:
One of the goals for this window is to have  content not move as states change. Can we always have this dropdown placed there, but just make it disabled when using other providers (by setting GUI.enabled = false and then back to true at the end)?